.buildkite/engineer

```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/1.33/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/add-runner-options/latest/$OS/engineer.gz" --output engineer.gz
```

Comment:
Reminder: This should probably be changed before merge when https://github.com/prisma/engineer/pull/66 is merged.